lib/hammer_cli/output/output.rb

```diff
@@ -25,13 +25,15 @@ def print_error(msg, details=nil, msg_params = {}, options = {})
 
     def print_record(definition, record)
       adapter.print_record(definition.fields, record) if appropriate_verbosity?(:record)
+      adapter.reset_context
```

Comment:
Why it is necessary to remove the :fields from context?

Reply:
I do it to be able to reset fields selection in case of using hammer in shell mode.